codegen/sections_test.go

```diff
@@ -0,0 +1,68 @@
+package codegen
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestHeader(t *testing.T) {
+	const (
+		pack  = "testpackage"
+		title = "test title"
+	)
+	var (
+		imprt   = []*ImportSpec{&ImportSpec{Path: "test"}}
+		imports = append(imprt, &ImportSpec{Path: "other"})
+	)
+	cases := map[string]struct {
+		Title    string
+		Imports  []*ImportSpec
+		Expected string
```

Comment:
They don't but I like to do it this way to make the field names stand out when defining the test cases.